src/main/java/com/google/cloud/spanner/pgadapter/utils/MutationWriter.java

```diff
@@ -72,25 +72,22 @@ public int getRowCount() {
     return this.rowCount;
   }
 
-  /** Build mutation to add to mutations list with data contained within a CopyData payload */
-  public void buildMutation(ConnectionHandler connectionHandler, byte[] payload) throws Exception {
-    List<CSVRecord> records = parsePayloadData(payload);
-    if (!records.isEmpty()
-        && !payloadFitsInCurrentBatch(records.size() * records.get(0).size(), payload.length)) {
-      rollback(connectionHandler, payload);
-      long mutationCount = this.mutationCount + records.size() * records.get(0).size();
-      long commitSize = this.batchSize + payload.length;
+  public void addMutations(ConnectionHandler connectionHandler, byte[] payload) throws Exception {
```

Comment:
can you name this addCopyData since we aren't doing anything with mutations yet

Reply:
Done.